include/lmms_constants.h

```diff
@@ -25,59 +25,27 @@
 #ifndef LMMS_CONSTANTS_H
 #define LMMS_CONSTANTS_H
 
+#include "lmmsconfig.h"
+#include "LmmsTypes.h"
+
 namespace lmms
 {
-
 // Prefer using `approximatelyEqual()` from lmms_math.h rather than
 // using this directly
 inline constexpr float F_EPSILON = 1.0e-10f; // 10^-10
 
+inline constexpr ch_cnt_t DEFAULT_CHANNELS = 2;
```

Comment:
I know you copied this name from elsewhere, but shouldn't constants like this normally be in capitals?

Reply:
Whoops nevermind, this is a type.  Resolving.

Reply:
Nah, it is a constant. Good catch tho

Reply:
But changing that will bloat the diff further and I'm trying to minimise, should I?

Reply:
As written, this was a misread of the diff.  Please disregard.